lib/features/reps_core/reps_admin/reps_admin.module

```diff
@@ -413,3 +413,22 @@ function reps_admin_theme_registry_alter(&$theme_registry) {
     }
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function reps_admin_form_alter(&$form, &$form_state, $form_id) {
+  // views_form_core_content_administration_page.
+  if ($form_id == 'views_exposed_form') {
```

Comment:
check the #form['name'] or something to identify the form (so it does not apply to others)